app/controllers/api/admin/sessions_controller.rb

```
@@ -3,18 +3,15 @@ class Api::Admin::SessionsController < Devise::SessionsController
   skip_before_action :verify_authenticity_token
 
   def create
+    abc = 123
```

rubocop (1.68.0)

[warning] Useless assignment to variable - `abc`.

(Lint/UselessAssignment)